trigger/version.go

```diff
@@ -170,6 +170,13 @@ func (t *versionTriggers) versionOutcome(sub *event.Subscription) (*notification
 		return nil, nil
 	}
 
+	isReady, err := t.waitOnChildrenOrSiblings(sub)
```

Comment:
Similar to the comment in rest/data, it's still reasonable to subscribe to a version outcome by itself. I would consider logging a new type of patch event called something like PATCH_FAMILY_FINISHED which will simplify the handling you have to determine if the you need to send things with the patch outcome event

Reply:
I'm not sure I fully understand this comment. However, I would argue that any time a version has children, it is not considered to have an "outcome" unless the children are done. The children become part of the patch (similar to builds) and its status is based on it. Please elaborate more if I'm missing something.

Reply:
discussed over slack